util/json/unmarshal.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package json provides json utilities.
```

Comment:
JSON utilities, namely an Unmarshal wrapper that amortizes allocated garbage over subsequent runs.

or something

Reply:
Done